config/gAnalytics.js

```diff
@@ -0,0 +1,10 @@
+const { google } = require('googleapis')
+const clientEMail = process.env.CLIENT_EMAIL
+const privateKey = process.env.PRIVATE_KEY
+const scopes = ['https://www.googleapis.com/auth/analytics.readonly']
+
+module.exports = new google.auth.JWT({
+  email: clientEMail,
```

Comment:
Every user would have to add their own clientEMail and privateKey??

Reply:
@Rupeshiya We are using a service account to give all the users the access to the google analytics data.  An email is generated alongside a new client id, private key,  when a new service account is generated. Those details would be used here. https://cloud.google.com/iam/docs/service-accounts#differences_between_a_service_account_and_a_user_account